packages/mask/src/components/shared/PluginEnableBoundary.tsx

```diff
@@ -36,17 +38,24 @@ export const PluginEnableBoundary = memo<PluginEnableBoundaryProps>((props) => {
 
     if (disabled) {
         return (
-            <ActionButton
-                loading={loading}
-                startIcon={<Icons.Plugin size={18} />}
-                className={classes.root}
-                color="primary"
-                onClick={onEnablePlugin}
-                sx={{ mt: 10 }}>
-                <Typography fontSize={14} fontWeight={700}>
-                    {t('enable_plugin_boundary')}
-                </Typography>
-            </ActionButton>
+            <Stack alignItems="center">
+                <Stack justifyContent="center" alignItems="center" width="100%" boxSizing="border-box">
+                    <Typography fontWeight={400} fontSize={14}>
+                        {t('enable_plugin_boundary_description')}
+                    </Typography>
+                </Stack>
+                <ActionButton
+                    loading={loading}
+                    startIcon={<Icons.Plugin size={18} />}
+                    className={classes.root}
+                    color="primary"
+                    onClick={onEnablePlugin}
+                    sx={{ mt: 6 }}>
+                    <Typography fontSize={14} fontWeight={700}>
```

Comment:
No need to wrap `<Typography />` in `<ActionButton />`. Moreover, the default size of `<Typography />` is **14**.